### DISCUSSION TITLE
Implement partial override at specific size

### DISCUSSION THREAD

**Discussion by sbxchicago** [squidfunk/mkdocs-material#4915] [Q&A] [answered]
Hello, I'm curious if it is possible to use a partial to override a part of the theme at certain sizes. For example, to use a custom header at desktop sizes, but revert to the nav system provided by the Material theme at smaller viewports.

Thanks for the time.

**Comment by squidfunk** [comment, marked as answer]
Yes, that's possible. I suggest that you look at our navigation tabs implementation that is only mounted on screens above `1220px`. 

Specifically here:

https://github.com/squidfunk/mkdocs-material/blob/99eed0e810c09def8242d069e3d174ff5cc6f38d/src/partials/tabs.html#L26-L39

... and here:

https://github.com/squidfunk/mkdocs-material/blob/99eed0e810c09def8242d069e3d174ff5cc6f38d/src/assets/stylesheets/main/components/_tabs.scss#L44-L47

Note that `break-to-device` is a custom function that we defined. You could also resort to plain [media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries) for this.
